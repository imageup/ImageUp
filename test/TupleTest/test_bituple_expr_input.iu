function main() -> int
{
    int : x = 1;
    int : y = 2;
    float : a = 3.0;
    float : b = 4.0;
    tuple:tu = (x, y);
    tuple:tu1 = (a, b);
    printf(tu[0]);
    printf(tu[1]);
    printf(tu1[0]);
    printf(tu1[1]);
    prints("correct output should be: 1, 2, 3, 4");
    return 0;
}
