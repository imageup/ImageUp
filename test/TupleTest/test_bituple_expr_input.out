1
2
3
4
correct output should be: 1, 2, 3, 4
